dist/types/index.d.ts

```diff
@@ -0,0 +1,58 @@
+import { FocusOverlayOptions } from './types';
+import './styles.scss';
```

Comment:
This might help:

```
// compile-css.js

const fs = require('fs');
const sass = require('sass');
const packageImporter = require('node-sass-package-importer');

const EXPORT_DIR = 'dist';

const ENTRY_FILE = 'src/styles/styles.scss';
const OUT_FILE = 'dist/focusoverlay.css';

const checkExportDir = () => {
  if (!fs.existsSync(EXPORT_DIR)){
    fs.mkdirSync(EXPORT_DIR);
  }
};

sass.render({
  file: ENTRY_FILE,
  importer: packageImporter(),
  outFile: OUT_FILE,
  // ADD YOUR OPTIONS HERE?
}, function(error, result) {
  if (!error) {
    checkExportDir();

    fs.writeFile(OUT_FILE, result.css, function(err) {
      if (!err) {
        console.log('CSS File created');
      } else {
        console.error(err);
      }
    });
  } else {
    console.error(error);
  }
});
```

Reply:
thanks that worked